tools/pdp-metadata/pdp-metadata.js

```diff
@@ -37,6 +37,42 @@ async function performCatalogServiceQuery(config, query, variables) {
   return queryResponse.data;
 }
 
+function getJsonLd(product) {
+  const amount = product.priceRange?.minimum?.final?.amount || product.price?.final?.amount;
+  const brand = product.attributes.find((attr) => attr.name === 'brand');
+
+  const schema = {
+    '@context': 'http://schema.org',
+    '@type': 'Product',
+    name: product.name,
+    description: product.meta_description,
+    image: product['og:image'],
+    offers: [],
+    productID: product.sku,
+    sku: product.sku,
+    url: product.path,
+    '@id': product.path,
+  };
+
+  if (brand?.value) {
+    product.brand = {
+      '@type': 'Brand',
+        name: brand?.value,
+    };
+  }
+
+  if (amount?.value && amount?.currency) {
+    schema.offers.push({
+      '@type': 'http://schema.org/Offer',
```

Comment:
It's fine to just use `Offer` here as the `@type` attribute already defines the context of a valid schema.org type.